requirements.txt

```diff
@@ -4,4 +4,7 @@
 
 oslo.config>=1.4.0  # Apache-2.0
 sqlalchemy
-pyOpenSSL>=0.13.0,<=0.15.1
\ No newline at end of file
+pyOpenSSL>=0.13.0,<=0.15.1
+Click>=5.1
+requests
+python-neutronclient<2.5.0,>=2.4.0
```

Comment:
>= 2.4.0 will be a problem for Juno.

BTW, I'm not sure that we want to add a neutron dependency to apicapi. Perhaps, the neutron-sync command is better suited in the ml2-driver code.

Also, please update the RPM packaging information with additional dependencies ("Requires: " lines in rpm/apicapi.spec.in)